frontend/public/components/operator-lifecycle-manager/clusterserviceversion-resource.tsx

```diff
@@ -114,6 +114,19 @@ export const ClusterServiceVersionResourcesPage = connect(inFlightStateToProps)(
       : <StatusBox loaded={true} EmptyMsg={EmptyMsg} />;
   });
 
+export const ProvidedAPIPage = connect(inFlightStateToProps)((props: ProvidedAPIPageProps) => {
```

Comment:
New component for just showing a single "owned" CRD for an Operator, instead of multiple.